app/controllers/tag_controller.rb

```diff
@@ -287,25 +287,33 @@ def create
     node = Node.find nid
     tagnames.each do |tagname|
       # this should all be done in the model:
-
+      tagname = tagname.strip
       if Tag.exists?(tagname, nid)
         @output[:errors] << I18n.t('tag_controller.tag_already_exists')
+
+      elsif tagname.include?(":") && tagname.split(':').length < 2 # cant tag empty power tags
+        if tagname.split(':')[0] == "barnstar" || tagname.split(':')[0] == "with"
```

Comment:
hey @CleverFool77, I think we can implement this for all power tags, we have many - https://publiclab.org/wiki/power-tags. What do you say?

Reply:
Hi @gauravano In other power tags, If I'll add some of the other power tags like `build:`, `alert:` ,`notes:` etc then neither it shows traceback error nor its gets added, which is correct.

Only `with:` and `barnstars:` which were implemented in different way were showing traceback error and That's why I added only both of them here.
Thanks !!!

Reply:
Ok! I think, we can discuss it after merging this that whether we want to show alert if someone try to use incomplete power tag, even in case where no error is thrown. Thanks!